README-CN.md

```diff
@@ -0,0 +1,203 @@
+# app
+
+[![Build Status](https://travis-ci.org/murlokswarm/app.svg?branch=master)](https://travis-ci.org/murlokswarm/app)
+[![Go Report Card](https://goreportcard.com/badge/github.com/murlokswarm/app)](https://goreportcard.com/report/github.com/murlokswarm/app)
+[![Coverage Status](https://coveralls.io/repos/github/murlokswarm/app/badge.svg?branch=master)](https://coveralls.io/github/murlokswarm/app?branch=master)
+[![awesome-go](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/avelino/awesome-go#readme)
+[![GoDoc](https://godoc.org/github.com/murlokswarm/app?status.svg)](https://godoc.org/github.com/murlokswarm/app)
+
+一个使用
+[Go](https://golang.org), [HTML](https://en.wikipedia.org/wiki/HTML5),
+[CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)构建的多平台UI框架
```

Comment:
Looks like there is an encoding problem here.
<img width="374" alt="screen shot 2018-09-12 at 12 03 48 am" src="https://user-images.githubusercontent.com/2692731/45407807-63d5d680-b61f-11e8-89ea-4aec8125fcb3.png">

Reply:
done